.devcontainer/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.22.1-bookworm
+FROM golang:1.22.2-bookworm
```

Comment:
In this case, the `t.Cleanup()` function should be used to reserve post-processing instead of using the `defer` statement because the `cleanup` function reserved by the `defer` statement will be executed before finishing sub test functions.

Reply:
It is recommended to insert the `t.Parallel()` function to run the top-level test function in parallel.